slasher/db/iface/interface.go

```diff
@@ -31,6 +31,9 @@ type ReadOnlyDatabase interface {
 	LatestIndexedAttestationsTargetEpoch(ctx context.Context) (uint64, error)
 
 	// MinMaxSpan related methods.
+	EpochSpans(ctx context.Context, epoch uint64) ([]byte, error)
+	SetValidatorSpan(ctx context.Context, spans []byte, validatorIdx uint64, newSpan detectionTypes.Span) ([]byte, error)
+	GetValidatorSpan(ctx context.Context, spans []byte, validatorIdx uint64) (detectionTypes.Span, error)
```

Comment:
```suggestion
	GetValidatorSpan(ctx context.Context, validatorIdx uint64, spans []byte) (detectionTypes.Span, error)
```

Do you mind swapping the parameters for these 2 functions? Usual pattern is (ctx, idx, data)

Reply:
fixed